template/readme/acsf-setup.md

```diff
@@ -1,6 +1,15 @@
 # ACSF Setup
 
 To configure a project to run on ACSF, perform the following steps after initially setting up Bolt:
- 
+
+1. Add a `hosting` variable set to `acsf` in `project.yml`
 1. Execute `./bolt.sh acsf:init` from the project root.
 1. Add the acsf module as a dependency to your installation profile.
+
+## Factory Hooks and settings.php
+
+Acquia Cloud Site Factory does not allow changes to `settings.php`. In order to make additions normally done with `settings.php`, factory hooks are used.
```

Comment:
It seems like we may want to include example scripts for pre-settings-php and post-settings-php. I'd be interested in how Bolt generated projects on ACSF are using these. I'd imagine at least a script with a require statement for `cache.settings.php`.